articles/ams-team-setups.md

```diff
@@ -0,0 +1,43 @@
+---
+authorName: Tijana Krstajic, Guido Angenendt
+authorGithubUsername: doogu
+issue: 91
+title: Operations Team Setups
```

Comment:
Can we make the title more Zühlke Focused?

Or the article less Zühlke focused: AMS, ADM,...
Inspiration for DevOps antipatterns can be found on https://web.devopstopologies.com/

Reply:
AMS is a generally used term, not Zuehlke centric. ADM potentially we could eliminate, but that would be negating the Zuehlke project best practices...